Source/bmalloc/libpas/src/libpas/pas_probabilistic_guard_malloc_allocator.h

```diff
@@ -53,8 +53,14 @@ struct pas_pgm_storage {
 // including guard pages and wasted memory
 #define PAS_PGM_MAX_VIRTUAL_MEMORY (1024 * 1024 * 1024)
 
-// Probability that we should call PGM in percentage (0-100)
-#define PAS_PGM_PROBABILITY (1)
+// Probability that we should call PGM is 0.1%.
+// PGM is only enabled on macOS for now. Other platforms should be considered in the future.
+#if PAS_PLATFORM(MAC)
+#define PAS_SHOULD_ALLOC_USING_PGM (PAS_UNLIKELY(pas_get_fast_random(1000) < 1))
+#else
+#define PAS_SHOULD_ALLOC_USING_PGM (0)
+#endif
```

Comment:
Let's make it a bit clear that this is activation rate, not a rate used by allocation.
I mean, activation is one time check. And if it is not activated, then PGM is not used at all.
If it is activated, PGM will be used (this is the current libmalloc's policy).